applications/nrf_desktop/VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 2
```

Comment:
I think we need to ensure that versions for B0 are aligned too (before NCS 2.8). If you prefer to do it outside of this PR, then please open a follow-up ticket.

Reply:
I think we agreed that B0 will not follow this versioning scheme as it uses one number for versioning. If you have any idea how to approach this, please leave a note in the ticket:

https://nordicsemi.atlassian.net/browse/NCSDK-28603

Reply:
I have no exact idea yet, but we may think about encoding `VERSION` file content on one number somehow.

Reply:
Will this change have any impact on the documentation?

Reply:
Well, it would be worth mentioning the change in the documentation: if we manage to make the B0 version number be derived somehow from the `VERSION` file.